PerformanceCalculator/Simulate/ManiaSimulateCommand.cs

```diff
@@ -80,11 +80,12 @@ public class ManiaSimulateCommand : SimulateCommand
             int delta = targetTotal - remainingHits;
 
             // Each great and perfect increases total by 5 (great-meh=5)
-            // There is no difference in accuracy between them, so just halve arbitrarily.
-            greats = Math.Min(delta / 5, remainingHits) / 2;
-            int perfects = greats.Value;
+            // There is no difference in accuracy between them, so just halve arbitrarily (favouring perfects for an odd number).
+            int greatsAndPerfects = Math.Min(delta / 5, remainingHits);
+            greats = greatsAndPerfects / 2;
+            int perfects = greats.Value + (greatsAndPerfects % 2);
```

Comment:
I would have done

```suggestion
            int perfects = greatsAndPerfects - greats.Value;
```

to dispel any and all doubts of inaccuracy. I guess what you have works but feels needlessly contrived?

Reply:
Personally I think divmod is a little more clear since the fact that it's integer division isn't immediately obvious.
But simply subtracting from that value is certainly more readable given c# doesn't have a built in divmod function.
I'll change it tonight :ok_hand: